client/src/pages/pitchDoc/PitchDoc.tsx

```diff
@@ -301,11 +328,49 @@ function PitchDoc(): ReactElement {
 
       <div className="content-wrapper">
         <div className="top-section">
-          <div className="pitchdoc-title">The Pitch Doc</div>
+          <div className="pitchdoc-title">
+            {isAdmin ? 'Pitch Approval' : 'The Pitch Doc'}
+          </div>
+
+          {isAdmin ? (
+            <div className="buttons-section">
+              <Button
```

Comment:
So right now these buttons change the UI tabs but don't seem to make any changes to the `displayedCards`. I don't know if there is a block so please let me know if there is one, but you will probably need a hook that has currentTab as a dependency and updates the displayedCards depending on which tab is showing